core/Updates/4.5.0-b1.php

```diff
@@ -29,11 +29,16 @@ public function __construct(MigrationFactory $factory)
 
     public function getMigrations(Updater $updater)
     {
-        $migration1 = $this->migration->db->changeColumnType('session', 'data', 'MEDIUMTEXT');
+        $migrations = [];
+        $migrations[] = $this->migration->db->changeColumnType('session', 'data', 'MEDIUMTEXT');
+
+        //database using utf8mb4 set row format dynamic
+        if (DbHelper::getDefaultCharset() == "utf8mb4") {
+            $migrations[] = $this->migration->config->set('disable_dynamic_row_format', false);
```

Comment:
fyi similar to above comment we usually would use here `0` instead of `false` for these config values.